src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx

```diff
@@ -0,0 +1,284 @@
+2. Select your Unity app. 
+3. Go to **Settings > Application** and copy the displayed **Application token**.
+ 
+
+You'll add this application token in the next step.
+</Step>
+<Step>
+
+In your Unity IDE, click Tools → NewRelic → Getting Started to open the NewRelic editor window.
+
+<img
+  title="Unity AppToken Setup"
+  alt="Add App Token in Unity Apps"
+  src={unityAgentAppTokenSetup}
+  style={{ maxWidth: '70%' }}
+/>
+Make sure you paste your application token(s) into `appToken = ""` in the code above. If you deployed your Unity app to both iOS and Android platforms, you'll need to add two separate tokens: one for iOS and one for Android.
+</Step>
+<Step>
+### (Android-only) Configure your Android app [#configure-android]
+
+Customize Gradle Templates
+If using Unity 2019 or later, add the following to your Gradle files:
+1. Include the New Relic Maven repository URL in the Gradle build settings. To do this, open your `mainTemplate.gradle` file (usually located in Assets/Plugins/Android folder) and add the New Relic Maven URL like this:
+```groovy
+   allprojects {
+    buildscript {
+        repositories {**ARTIFACTORYREPOSITORY**
+            google()
+            jcenter()
+            mavenCentral()
+        }
+    }
+```
+
+2. Add the New Relic classpath to your project-level `baseProjectTemplate.gradle` file (typically located in the android folder in your Unity project):
+
+```groovy
+        dependencies {
+            // If you are changing the Android Gradle Plugin version, make sure it is compatible with the Gradle version preinstalled with Unity
+            // See which Gradle version is preinstalled with Unity here https://docs.unity3d.com/Manual/android-gradle-overview.html
+            // See official Gradle and Android Gradle Plugin compatibility table here https://developer.android.com/studio/releases/gradle-plugin#updating-gradle
+            // To specify a custom Gradle version in Unity, go do "Preferences > External Tools", uncheck "Gradle Installed with Unity (recommended)" and specify a path to a custom Gradle version
+            classpath 'com.newrelic.agent.android:agent-gradle-plugin:7.2.0'
+            **BUILD_SCRIPT_DEPS**
+        }
+    }
+
+ ```
+3. 2. Apply the `newrelic` plugin to the top of your `launcherTemplate.gradle` file(typically located in the android folder in your Unity project):
+
+```groovy
+apply plugin: "com.android.application"
+apply plugin: 'newrelic' // <-- add this
+```
+4. Add `INTERNET` and `ACCESS_NETWORK_STATE` permissions in your `AndroidManifest.xml` file:
+
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+```
+</Step>
+
+<Step>
+### (iOS-only) Please ensure that your External Dependency Manager settings match the following configuration. 
+
+In your Unity IDE, navigate to Assets → External Dependency Manager → iOS Resolver → Settings: [#configure-ios]
```

Comment:
@ndesai-newrelic What does a user have to look for under settings? Or what fields do they have to configure? Just having the screenshot is not enough I don't think

Reply:
basically customers need to follow this pod configurations to resolve ios dependency.
<img width="476" alt="Screenshot 2023-12-13 at 11 15 38 AM" src="https://github.com/newrelic/docs-website/assets/89222514/39263127-d5c8-4c1d-8c0e-0d46b782ea29">

Reply:
customer need to add podfile to the main target always.